requirements.txt

```diff
@@ -7,4 +7,9 @@ pycodestyle==2.5.0
 pyflakes==2.1.1
 pytz==2019.2
 sqlparse==0.3.0
-djangorestframework~=3.10.2	
+djangorestframework~=3.10.2
+gunicorn==19.9.0
+whitenoise==4.1.3
+psycopg2-binary==2.8.3
+python-dotenv==0.10.3
+dj-database-url==0.5.0
```

Comment:
I read this as `DJ DATABASE IN THE HOUSSSSEEEEE` *bew bew bewwwwww*

Reply:
📣 📣